### ISSUE TITLE
[Screen Reader – Make code – Home page] H1 heading is not defined on ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3519]
**User Experience:**
Users using screen reader will not be able to identify the headings properly and navigate on the page efficiently if there is no heading level one defined on the page.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify whether H1 heading is defined on “Home Page” Page or not.

**Actual Result:**
H1 heading is not defined on ‘Home Page’.

**Expected Result:**
H1 heading should be defined on ‘Home Page’. ‘My Projects’ heading is define on H1 level.

**WCAG Reference:**
[WCAG 2.4.6 - Headings and Labels](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.6, #Win10-Edge(Chromium)

![WCAG2 4 6_H1 is not defined](https://user-images.githubusercontent.com/72871089/97663365-5b5ee500-1a9f-11eb-85bb-5c51b595dfee.jpg)
